### ISSUE TITLE
Clear Mock data on Checkout

### ISSUE THREAD

**Issue by amenk** [integer-net/magento2-shippingpreselection#3]
From the readme:

> Important: This is WIP as of yet - shipping address mock data still needs to be cleared when entering checkout.

Interesting enough we did not run in to this issue before with Luma checkout, but now we do, when we switched to OneStepCheckout.

We will try to provide a pull request - or did you @schmengler proceed on this issue already?

**Comment by lbuchholz** [comment]
Yes, there is a solution for that already. Please give us some time to update repo here.

**Comment by lbuchholz** [comment]
@amenk Repo has been updated to latest version. You should be able to extend config value (xml only) so the module will not fire, but even clear the mocked data upon entering checkout.
